app/client/apiconsole.ts

```diff
@@ -117,7 +117,11 @@ function setParamValue(resolvedParam: any, value: ParamValue) {
     // For every endpoint in the spec...
     for (const [pathKey, path] of spec.get("paths").entries()) {
       for (const [method, operation] of path.entries()) {
-
+        // Skip the $ref for now, it is only used in `scim` endpoints witch don't share
```

Comment:
```suggestion
        // Skip the $ref for now, it is only used in `scim` endpoints which don't share
```

![Gif of the Monty Python's Holy Grail yelling "witch"](https://media1.tenor.com/m/ZSxjbWZ4S7QAAAAd/monty-python-holy-grail-and-the-flying-circus-john-cleese.gif)